README.md

```diff
@@ -11,6 +11,17 @@ https://esi.tech.ccp.is/
 **Please do not open issues for anything security related here, send an email to security@ccpgames.com instead.**
 
 
+## Opening a new issue
+
+Please use the search function before opening a new issue.
```

Comment:
Maybe specifically remind people to search through closed issues as well?

Reply:
good call, added